### PR TITLE
Readme: new troubleshootings section

### PR DESCRIPTION
**Pull request by jotak**
@pilhuhn @tsegismont can you have a look? Any idea of other points to add?

You can see it there: https://github.com/jotak/hawkular-grafana-datasource/tree/troubleshooting#troubleshooting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,3 +71,21 @@ Once you have set some variables, you can use them in graph queries: either for
...
+
+* Check your browser's javascript debugging tool. If you see an error mentioning issues with CORS, switch to `proxy` mode in the datasource configuration.
+
+### I can't query by tag, the option is not displayed
+
+Querying by tag was introduced before the plugin was properly versioned, so if you have a version >= 1.0.2 you should have it. However it is only enabled when it talks to hawkular-metrics >=0.20.0. To check your version of hawkular-metrics just open its status page in a browser (`[host]/hawkular/metrics` or `[host]/hawkular/metrics/status`).
+
+### I'm running Hawkular in OpenShift, connection is OK but I can't query any metric
+
+Check your version of hawkular-metrics (`[host]/hawkular/metrics` or `[host]/hawkular/metrics/status`). Prior to 0.16.0, metric names containing slashes, like in OpenShift, were unfortunately not showing up in Grafana; you could consider upgrading hawkular-metrics.
```
Comment (tsegismont) [inline README.md]: If you suggest upgrading H-Metrics, add a link to help the user finding how.

File: README.md [hunk 2]
```diff
@@ -71,3 +71,21 @@ Once you have set some variables, you can use them in graph queries: either for
...
+
+* Make sure any credential / token matches your installation. In general, if you installed a standalone hawkular-metrics server without any specific configuration you probably don't have any authentication information to provide. If you installed hawkular-services using its [installation guide](http://www.hawkular.org/hawkular-services/docs/installation-guide/) you will probably have to fill-in the basic auth fields. If you are using Hawkular from OpenShift, you have to provide a Bearer token in the `Token` field. Tokens can be [generated temporarily](https://docs.openshift.com/enterprise/3.1/architecture/additional_concepts/authentication.html) (got to `[OpenShift host]/oauth/token/request`) or from a [Service account](https://docs.openshift.com/container-platform/3.3/rest_api/index.html#rest-api-serviceaccount-tokens) in OpenShift.
+
+* Check your browser's javascript debugging tool. If you see an error mentioning issues with CORS, switch to `proxy` mode in the datasource configuration.
```
Comment (tsegismont) [inline README.md]: Isn't proxy mode the default?

Comment (jotak) [inline README.md]: It's what is recommended in our doc, but the default is direct and we don't have control on that. There's been several people doing this mistake recently, so it worth saying that again on more time :)

File: README.md (hunk 1 above)
Comment (jotak) [inline README.md]: Well, actually I'm not sure how to upgrade metrics in openshift... especially if it's enterprise with  already enabled metrics

**Comment by tsegismont** [comment]
Maybe @mwringe knows where you could point users to. If we don't know how
to upgrade Origin Metrics then we should not suggest to do it.

2016-12-09 11:03 GMT+01:00 Joel Takvorian <notifications@github.com>:

**Comment by pilhuhn** [comment]
You also should mention that the tenant is case sensitive. Hawkular != hawkular

**Comment by jotak** [comment]
Updated with your remarks

**Comment by mwringe** [comment]
Updating Hawkular Metrics is described in the OpenShift documentation (https://docs.openshift.org/latest/install_config/upgrading/manual_upgrades.html#manual-upgrading-cluster-metrics)

**Comment by jotak** [comment]
Thanks @mwringe , I'll update with that link

**Pull request merged**